.github/workflows/odh-build-and-publish-operator-image.yaml

```diff
@@ -4,8 +4,12 @@
 name: ODH
 
 on:
-  - push
-  - pull_request
+  push:
+    branches:
```

Comment:
Should it also be triggered on tag?

Reply:
you are right, there should be `tags` under push actions, let me update it